User/admin.php

```diff
@@ -13,6 +13,20 @@
         <?php include("../Global/navBar.php"); ?>
         <?php if (!isset($_SESSION['userLoggedIn']) || $userLoggedIn->getLevel() < 3) header("Location: ../Global/forbidden.php"); ?>
 
+    <div class="container d-flex justify-content-center">
+        <form class="form-inline my-2 my-lg-0">
+            <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+        </form>
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Active/Inactive
+        </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item" href="#">Action</a>
+            <a class="dropdown-item" href="#">Another action</a>
+        <div class="dropdown-divider"></div>
+            <a class="dropdown-item" href="#">Something else here</a>
```

Comment:
Use Select HTML element, much cleaner that way, Default should be active, and another option should be In active

Reply:
Fixed